flexget/plugins/output/qbittorrent.py

```diff
@@ -77,6 +88,46 @@ def prepare_config(self, config):
         config.setdefault('fail_html', True)
         return config
 
+    def add_entries(self, task, config):
+        for entry in task.accepted:
+            formdata = {}
+
+            savepath = entry.get('path', config.get('path'))
+            if savepath:
+                formdata['savepath'] = savepath
+
+            label = entry.get('label', config['label']).lower()
+            if label:
+                formdata['label'] = label # qBittorrent v3.3.3-
+                formdata['category'] = label # qBittorrent v3.3.4+
+
+            downloaded = not entry['url'].startswith('magnet:')
+
+            if task.manager.options.test:
+                log.info('Test mode.')
+                log.info('Would add torrent to qBittorrent with:')
+                if downloaded:
+                    log.info('    File: %s', entry.get('file'))
+                else:
+                    log.info('    Url: %s', entry.get('url'))
+                log.info('    Save path: %s', formdata.get('savepath'))
+                log.info('    Label: %s', formdata.get('label'))
+                continue
+
+            if downloaded:
+                if 'file' not in entry:
+                    entry.fail('File missing?')
+                    continue
+                if not os.path.exists(entry['file']):
+                    tmp_path = os.path.join(task.manager.config_base, 'temp')
+                    log.debug('entry: %s' % entry)
```

Comment:
Logger module can handle string formatting, so change the % to a comma. You already did it correctly in the --test block.